packages/radio-group/src/Radio.tsx

```diff
@@ -314,20 +360,43 @@ function Radio({
           focusTargetElement={inputElement}
           className={cx(radioBoxStyle, radioBoxSize[normalizedSize])}
           borderRadius="100%"
+          color={!darkMode ? { focused: 'transparent' } : undefined} // TODO: Refresh - overriding the focus style for light mode
```

Comment:
Small nit: personally I'd flip the ternary around. Not a fan of checking for falseness